tooling/maven/camel-package-maven-plugin/src/main/java/org/apache/camel/maven/packaging/EndpointDslMojo.java

```diff
@@ -480,16 +509,12 @@ private void createEndpointDsl(String packageName, ComponentModel model, List<Co
             method.getJavaDoc().setText(desc);
         }
 
-        String fileName = packageName.replaceAll("\\.", "\\/") + "/" + builderName + "Factory.java";
-        writeSourceIfChanged(javaClass, fileName, false);
+        writeSourceIfChanged(javaClass, packageName.replaceAll("\\.", "\\/"), builderName + "Factory.java", false);
```

Comment:
I guess this could use `String#replace` instead of `String#replaceAll` to avoid the regex and therefore improve performance. (Not sure if this is relevent.)